.lycheeignore

```diff
@@ -2,3 +2,5 @@ http://localhost
 http://jaeger-collector
 https://github.com/open-telemetry/opentelemetry-go/milestone/
 https://github.com/open-telemetry/opentelemetry-go/projects
+file:///home/runner/work/opentelemetry-go/opentelemetry-go/libraries
+file:///home/runner/work/opentelemetry-go/opentelemetry-go/manual
```

Comment:
This seems specific to your local setup. Would there be a way for you to setup a global lycheeignore file?

Reply:
No, the build of this PR was failing earlier because of check-links error, https://github.com/open-telemetry/opentelemetry-go/actions/runs/4238014997/jobs/7364617339:

> <img width="1360" alt="image" src="https://user-images.githubusercontent.com/4140793/220615272-7e1b0bb0-bbeb-492f-80f0-8c00a92af551.png">

Reply:
Gotcha. I'd suggest adding a comment specifying this path is for GH Actions. But it seems lycheeignore doesn't allow comments 😞